pkg/migration/controllers/migrationschedule.go

```diff
@@ -107,7 +111,89 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 
 		return nil
 	}
+	migrationSchedule.Spec = setScheduleDefaults(migrationSchedule.Spec)
+	if migrationSchedule.GetAnnotations() != nil {
+		if _, ok := migrationSchedule.GetAnnotations()[StorkMigrationAnnotation]; ok {
+			// check status of all migrated app in cluster
+			logrus.Infof("Migration schedule is on dr cluster")
+			isActivated, err := getMigratedAppStatus(migrationSchedule)
+			if err != nil {
+				return err
+			}
+			migrationSchedule.Status.ApplicationActivated = isActivated
+			msg := fmt.Sprintf("Setting AppActive status to: %v", isActivated)
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				"AppsActivated",
+				msg)
+			log.MigrationScheduleLog(migrationSchedule).Warn(msg)
+			return m.client.Update(context.TODO(), migrationSchedule)
+		}
+	}
+	if !(*migrationSchedule.Spec.Suspend) {
+		remoteConfig, err := getClusterPairSchedulerConfig(migrationSchedule.Spec.Template.Spec.ClusterPair, migrationSchedule.Namespace)
+		if err != nil {
+			return err
+		}
+		remoteOps, err := storkops.NewForConfig(remoteConfig)
+		if err != nil {
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				string(stork_api.MigrationStatusFailed),
+				err.Error())
+			return nil
+		}
+		coreOps, err := core.NewForConfig(remoteConfig)
+		if err != nil {
+			m.recorder.Event(migrationSchedule,
+				v1.EventTypeWarning,
+				string(stork_api.MigrationStatusFailed),
+				err.Error())
+			return nil
+		}
+		migrSched, err := remoteOps.GetMigrationSchedule(migrationSchedule.Name, migrationSchedule.Namespace)
+		if errors.IsNotFound(err) {
+			// TODO:generate event?
+			namespace, err := core.Instance().GetNamespace(migrationSchedule.Namespace)
+			if err != nil {
```

Comment:
handle ErrNotFound() ? This code could race with Migration controller which tries to create a namespace as a part of the Migration